notification/src/utils/commons.js

```diff
@@ -18,7 +18,22 @@ function sendResponse(response, statusCode = 200, headers, data) {
   response.end(data)
 }
 
+function getNotificationForTracking(notification) {
+  if (notification && notification.NotificationRequestItem) {
+    const notificationRequestItem = notification.NotificationRequestItem
+    return {
+      eventCode: notificationRequestItem.eventCode,
+      eventDate: notificationRequestItem.eventDate,
+      pspReference: notificationRequestItem.pspReference,
+      success: notificationRequestItem.success,
+      // reason: notificationRequestItem.reason // not sure about that.
```

Comment:
Can we uncomment this?

Reply:
It can be not existing... do you think it makes sense to put it?

Reply:
dead3ff